apis/core/lib/domain/column-mapping/create.ts

```diff
@@ -0,0 +1,36 @@
+import { GraphPointer } from 'clownface'
+import { rdf, schema } from '@tpluscode/rdf-ns-builders'
+import { cc } from '@cube-creator/core/namespace'
+import { ResourceStore } from '../../ResourceStore'
+import * as id from '../identifiers'
+import { resourceStore } from '../resources'
+import { NamedNode } from 'rdf-js'
+import $rdf from 'rdf-ext'
+
+interface CreateColumnMappingCommand {
+  table: GraphPointer<NamedNode>
+  column: GraphPointer
+  store?: ResourceStore
+}
+
+export function createColumnMapping({
+  table,
+  column,
+  store = resourceStore(),
+}: CreateColumnMappingCommand): GraphPointer {
+  const columnName = column.out(schema.name).value!
+
+  const columnMapping = store.create(id.columnMapping(table, columnName))
+
+  columnMapping
+    .addOut(rdf.type, cc.ColumnMapping)
+    .addOut(cc.sourceColumn, column)
+    .addOut(cc.targetProperty, defaultProperty(columnName))
+
+  return columnMapping
```

Comment:
Now you're probably missing a `store.save()` somewhere. Look at what Luca did in #129, I think it's probably a good pattern